src/main/java/io/druid/data/input/MapBasedRow.java

```diff
@@ -164,11 +151,8 @@ public boolean equals(Object o)
     if (!event.equals(that.event)) {
       return false;
     }
-    if (!timestamp.equals(that.timestamp)) {
-      return false;
-    }
+    return timestamp.equals(that.timestamp);
```

Comment:
Its funny that you say that. I imported the IntelliJ-Settings from the druid project. And it refactored the code in that way. I did not want to change that originally but then thought, why not.